pkg/caret/R/caret-package.R

```diff
@@ -0,0 +1,1758 @@
+#' quantitative structure-activity and structure-selectivity relationships of
+#' dihydrofolate reductase inhibitors, \emph{Journal of Computer-Aided
+#' Molecular Design}, Vol. 18, pg. 309--331.
+#' @keywords datasets
+NULL
+
+
+
+
+
+#' German Credit Data
+#'
+#' Data from Dr. Hans Hofmann of the University of Hamburg.
+#'
+#' These data have two classes for the credit worthiness: good or bad. There
+#' are predictors related to attributes, such as: checking account status,
+#' duration, credit history, purpose of the loan, amount of the loan, savings
+#' accounts or bonds, employment duration, Installment rate in percentage of
+#' disposable income, personal information, other debtors/guarantors, residence
+#' duration, property, age, other installment plans, housing, number of
+#' existing credits, job information, Number of people being liable to provide
+#' maintenance for, telephone, and foreign worker status.
+#'
+#' Many of these predictors are discrete and have been expanded into several
+#' 0/1 indicator variables
+#'
+#'
+#' @name GermanCredit
+#' @docType data
+#' @source UCI Machine Learning Repository
+#' @keywords datasets
+NULL
+
+
+
+
+#' Multidrug Resistance Reversal (MDRR) Agent Data
+#'
+#' Svetnik et al. (2003) describe these data: "Bakken and Jurs studied a set of
+#' compounds originally discussed by Klopman et al., who were interested in
+#' multidrug resistance reversal (MDRR) agents. The original response variable
+#' is a ratio measuring the ability of a compound to reverse a leukemia cell's
+#' resistance to adriamycin. However, the problem was treated as a
+#' classification problem, and compounds with the ratio >4.2 were considered
+#' active, and those with the ratio <= 2.0 were considered inactive. Compounds
+#' with the ratio between these two cutoffs were called moderate and removed
+#' from the data for twoclass classification, leaving a set of 528 compounds
+#' (298 actives and 230 inactives). (Various other arrangements of these data
+#' were examined by Bakken and Jurs, but we will focus on this particular one.)
+#' We did not have access to the original descriptors, but we generated a set
+#' of 342 descriptors of three different types that should be similar to the
+#' original descriptors, using the DRAGON software."
+#'
+#' The data and R code are in the Supplemental Data file for the article.
+#'
+#'
+#' @name mdrr
+#' @aliases mdrr mdrrClass mdrrDescr
+#' @docType data
+#' @return \item{mdrrDescr}{the descriptors} \item{mdrrClass}{the categorical
+#' outcome ("Active" or "Inactive")}
+#' @source Svetnik, V., Liaw, A., Tong, C., Culberson, J. C., Sheridan, R. P.
+#' Feuston, B. P (2003).  Random Forest: A Classification and Regression Tool
+#' for Compound Classification and QSAR Modeling, \emph{Journal of Chemical
+#' Information and Computer Sciences}, Vol. 43, pg. 1947-1958.
+#' @keywords datasets
+NULL
+
+
+
+
+
+#' A List of Available Models in train
```

Comment:
The list of available models is automatically generated by a file in the `release_process` directory . Can you come up with a way to use that code to  work with Roxygen?